### ISSUE TITLE
Overlay doesn't go away

### ISSUE THREAD

**Issue by tomodutch** [TomSchillemans/Pipapi#1]
Whenever I click on either "Sign up" or "Sign in" in the navigation. A modal appears.
On closing the modal. The overlay doesn't disappear.

**Comment by TomSchillemans** [comment]
Yeah, that happens sometimes, but when you refresh the page, or if you go to another page (only possible when logged in) it doesn't happen. I am not sure what this causes.
